wsgi/odyssy/odyssy/urls.py

```diff
@@ -18,6 +18,7 @@
 
 urlpatterns = [
     url(r'^admin/?', include(admin.site.urls)),
-    url(r'^events/', include('events.urls')),
     url(r'^', include('basic.urls')),
+    url(r'^news/', include('news.urls')),
+    url(r'^events/', include('events.urls')),
```

Comment:
I was resolving the conflicts, I don't know how this is showing as my addition. 😕